warehouse/query-core/src/main/java/datawave/query/postprocessing/tf/FunctionReferenceVisitor.java

```diff
@@ -97,21 +99,20 @@ public Multimap<String,Function> functions() {
     @Override
     public Object visit(ASTFunctionNode node, Object data) {
         final int nChildren = node.jjtGetNumChildren();
-        if (nChildren < 3) {
-            log.error("Function node does have 3 children-- must supply a namespace, function name and at least one argument.");
+        if (nChildren < 2) {
```

Comment:
This can be simplified to `if(node.jjtGetNumChildren() < 2)`. Additionally, the error message needs to be corrected to say that the function node does **not** have 2 children.

Reply:
good point.  i wonder why it was written like that in the first place.  anyway... updated.